tests/testsuite/collisions.rs

```diff
@@ -52,13 +52,15 @@ fn collision_dylib() {
     // `j=1` is required because on Windows you'll get an error due to
     // two processes writing to the file at the same time.
     p.cargo("build -j=1")
-        .with_stderr_contains(&format!("\
+        .with_stderr_data(&format!("\
```

Comment:
Not a blocker. 
Just call it out so we can deal with it later. See https://github.com/rust-lang/cargo/pull/14068#discussion_r1640230311